ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -507,6 +507,11 @@ def test_source_ipahealthcheck_dogtag_ca_dogtagcertsconfigcheck(self):
         Testcase checks behaviour of check DogtagCertsConfigCheck in
         ipahealthcheck.dogtag.ca when tomcat config file is removed
         """
+        version = tasks.get_pki_version(self.master)
+        if version >= parse_version("11.5"):
+            pytest.skip("Skipping test for 11.5 pki version, since the "
+                        "check CADogtagCertsConfigCheck itself is skipped"
```

Comment:
Add a space after skipped otherwise the message will look like `Skipped: Skipping test for 11.5 pki version, since the check CADogtagCertsConfigCheck itself is skippedSee ipa-healthcheck ticket 317`